cdap-cli/src/main/java/co/cask/cdap/cli/commandset/ScheduleCommands.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.cli.commandset;
+
+import co.cask.cdap.api.schedule.ScheduleSpecification;
+import co.cask.cdap.cli.CLIConfig;
+import co.cask.cdap.cli.Categorized;
+import co.cask.cdap.cli.CommandCategory;
+import co.cask.cdap.cli.ElementType;
+import co.cask.cdap.cli.exception.CommandInputError;
+import co.cask.cdap.cli.util.AbstractCommand;
+import co.cask.cdap.cli.util.RowMaker;
+import co.cask.cdap.cli.util.table.Table;
+import co.cask.cdap.cli.util.table.TableRenderer;
+import co.cask.cdap.client.ScheduleClient;
+import co.cask.common.cli.Arguments;
+import co.cask.common.cli.Command;
+import co.cask.common.cli.CommandSet;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.google.inject.Inject;
+import com.google.inject.Injector;
+
+import java.io.PrintStream;
+import java.util.List;
+
+/**
+ * Schedule commands.
+ */
+public class ScheduleCommands extends CommandSet<Command> implements Categorized {
```

Comment:
Actually I see that there is no endpoint in ``ProgramLifecycleHttpHandler`` to get the specification of one schedule. But I still think that more info should be printed for each schedule. Maybe have in the list command, another column that contains the arguments of each specific schedule. I'm not sure how that should look like.